docs/pages/access-controls/access-requests/resource-requests.mdx

```diff
@@ -396,29 +421,35 @@ spec:
   deny: {}
 ```
 
-##### Access Requests for pods
+##### Access Requests for Kubernetes resources
 
-Teleport users can request access to a Kubernetes pod by running the following
+Teleport users can request access to a Kubernetes resources by running the following
 command:
 
 ```code
-$ tsh request create <Var name="pod-id" />
+$ tsh request create <Var name="resource-id" />
 ```
+###### Namespaced-scoped Resources
```

Comment:
Now that we're getting six heading levels deep, I think we should move the whole "Restrict the resources a user can request access to" H3 into its own reference guide.

Otherwise, the H6s don't render very nicely:

<img width="968" alt="image" src="https://github.com/gravitational/teleport/assets/25250668/f97dcc14-cba3-486f-bf62-36735a6ae570">


We can either do this as part of this PR or a separate one.

Reply:
I think it's better do it in a separate PR

Reply:
What would you think about extracting the "Access Requests for Kubernetes resources" H5 into an H3 below "Next steps"? We could put it above "Allow reviewers to see the hostnames of SSH Nodes". 

This way, we wouldn't need to use an H6 for any headings. I'd really like to avoid having an H6, since these are pretty much unreadable.